pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: csslib
-version: 1.0.0
+version: 1.0.1-wip
 description: A library for parsing and analyzing CSS (Cascading Style Sheets).
 repository: https://github.com/dart-lang/csslib
 
 topics:
   - css
 
 environment:
-  sdk: '>=2.19.0 <4.0.0'
+  sdk: ^3.0.0
```

Comment:
We should audit to see if we need any `final`, `interface`, or `base` classes while we are bumping the  version.

Reply:
I got no warnings 🤷

Reply:
Right, there are no warnings about being maximally permissive, but if any of the classes are explicitly not designed for extension or implementation then we'd be sending the wrong signal leaving everything at the default without usage restrictions.

Reply:
I don't *think* we're super worried about folks extending these types, right?

Reply:
I think the best practice would be to consider class modifiers with the sdk bump, but I agree it's probably ok to take a shortcut for this package.